src/modules/auction-house/CreateAuctionHouse.ts

```diff
@@ -0,0 +1,75 @@
+import { PublicKey, PublicKeyInitData, TransactionInstruction } from '@solana/web3.js';
```

Comment:
Its not createAuctionHouse we want for here its updateAuctionHouse. This file can be deleted.

Reply:
Sure